ex1-shai/checkNginx.sh

```diff
@@ -1,29 +1,29 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 which nginx > /dev/null 2>&1
 if [[ $? -eq 1 ]]; then	
     echo "Nginx is not currently installed on Linux, Begins installation."
-    sudo yum install -y nginx
-    sudo useradd --no-create-home nginx
-    sudo systemctl enable nginx
-    sudo systemctl start nginx
-    sudo systemctl is-active --quiet nginx && echo Nginx is running
+    yum install -y nginx
+    useradd --no-create-home nginx
+    systemctl enable nginx
+    systemctl start nginx
+    systemctl is-active --quiet nginx && echo Nginx is running
 else
     echo "Nginx already installed"
     echo "Did you interested in uninstall it? select the option that you prefer[Y/N]"
     read answer
     if [[ $answer = "Y" ]]; then
         echo "Start remove nginx."
-        sudo systemctl stop nginx
-        sudo systemctl disable nginx
-        sudo userdel -r nginx
+        systemctl stop nginx
+        systemctl disable nginx
+        userdel -r nginx
         
-        sudo rm -rf /etc/nginx
-        sudo rm -rf /var/log/nginx
-        sudo rm -rf /var/cache/nginx/
-        sudo rm -rf /usr/lib/systemd/system/nginx.servic
+        rm -rf /etc/nginx
```

Comment:
You can combine it into single `rm -rf` command:
~~~
rm -rf folder1/ folder2/ folder3/
~~~

Reply:
+1
Changed.